doc/man1/ciphers.pod

```diff
@@ -715,34 +715,34 @@ listed here because some ciphers were excluded at compile time.
 
 Verbose listing of all OpenSSL ciphers including NULL ciphers:
 
- openssl ciphers -v 'ALL:eNULL'
+ openssl ciphers -v "ALL:eNULL"
 
 Include all ciphers except NULL and anonymous DH then sort by
 strength:
 
- openssl ciphers -v 'ALL:!ADH:@STRENGTH'
+ openssl ciphers -v "ALL:!ADH:@STRENGTH"
```

Comment:
But you can't argue with suggestion that it would only be appropriate to choose expression that would *reduce the risk* of misunderstanding/misinterpretation in most popular choices. Besides, which shell doesn't honour single quotes? I mean you can say double quotes do fine with [my] shell X, but single quotes work with *all* of them.